db/methods.js

```diff
@@ -0,0 +1,51 @@
+const Comment = require("./model.js");
+
+const getComments = async (filter) => {
+  return Comment.find(filter);
+};
+
```

Comment:
Would it be better to have all as defaulted {}

Reply:
what do you mean by "all as defaulted"? I'm not sure what your suggestion is.

Reply:
Thanks for the thorough code review, David, much appreciated!

> Would it be better to have all as defaulted {}

what do you mean by "all as defaulted"? I'm not sure what your suggestion is.

Reply:
This one is my mistake if no parameter is passed it defaults as all or optionally passing an empty document. Thought you'd have to pass  an empty document.